src/assistant-markdown.css

```diff
@@ -0,0 +1,96 @@
+@import 'highlight.js/styles/github.css';
```

Comment:
Currently using the GitHub Markdown styling as a sensible out-of-the-box default for development/demonstration purposes.